.npmignore

```diff
@@ -0,0 +1,8 @@
+*.log
+.DS_Store
+.sass-cache
+build
+lib-cov
+node_modules
+vendor
+test
```

Comment:
This is `.gitignore` union `test`, because [npm docs](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package) says in absence of .npmignore, it takes .gitignore into account and when .npmignore is present, it essentially overrides .gitignore.